core/reflection.jvm/resources/META-INF/com.android.tools/proguard/kotlin-reflect.pro

```diff
@@ -0,0 +1,22 @@
+# When editing this file, update the following files as well:
+# - META-INF/com.android.tools/r8-from-1.6.0/kotlin-reflect.pro
+# - META-INF/com.android.tools/r8-upto-1.6.0/kotlin-reflect.pro
+# - META-INF/proguard/kotlin-reflect.pro
+# Keep Metadata annotations so they can be parsed at runtime.
+-keep class kotlin.Metadata { *; }
+
+# Keep implementations of service loaded interfaces
+-keep interface kotlin.reflect.jvm.internal.impl.builtins.BuiltInsLoader
+-keep class * implements kotlin.reflect.jvm.internal.impl.builtins.BuiltInsLoader { public protected *; }
+-keep interface kotlin.reflect.jvm.internal.impl.resolve.ExternalOverridabilityCondition
+-keep class * implements kotlin.reflect.jvm.internal.impl.resolve.ExternalOverridabilityCondition { public protected *; }
+
+# Keep generic signatures and annotations at runtime.
+# R8 requires InnerClasses and EnclosingMethod if you keepattributes Signature.
```

Comment:
Minor: this file is for Proguard specifically, but you mention R8 here

Reply:
Ah good catch, fixed a7a9f2b

Reply:
Also in `META-INF/proguard/kotlin-reflect.pro`

Reply:
That one is intended to be a generic fallback with configuration that could work with both at a minimal level, whereas these nested ones are more intended for tighter scopes where available